src/App.css

```diff
@@ -1,38 +1,80 @@
-.App {
-  text-align: center;
+body{
+margin: 0px;
+padding: 0px;
+overflow: hidden;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.main{
+    height: 100%;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    background-color: #BEFF00;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+
+.title{
+    font-size: 40px;
+    font-weight: 600;
+    color: #263200;
+    flex: 2;
+    padding-top: 20px;
+}
+
+.cnt-text{
```

Comment:
rename classes to proper full names and avoid use of short forms
:red_circle: cnt-text
:green_circle: counterText or counter-text
use descriptive and clear names to avoid confusion